.github/workflows/ruby.yml

```diff
@@ -0,0 +1,39 @@
+name: Ruby
```

Comment:
Well done for setting this up. Was it still on Travis, or no CI?

Reply:
No CI at all.